docs/notebooks/figanos_maps.ipynb

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #45.    # plt.savefig("images/station_map.png", bbox_inches='tight')
</pre><p>mais les images ne sont pas save  dans images mais dans _static/_gallery, non ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
They are, but if we were to enable this and then build the docs, we would see the same warnings as earlier. This line is only useful for developers.

Would it make sense to place these in their own hidden cells?

Reply:
Done!

Reply:
if it is in its own cell it doesn't save the figure from the previous cell. It saves an empty figure.